dao/src/main/java/com/epam/esm/creator/UserDetailsConverter.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.esm.creator;
+
+import com.epam.esm.entity.user.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserDetailsConverter {
+    public UserDetails convert(User user) {
+        return new org.springframework.security.core.userdetails.User(
+                user.getName(),
+                user.getPassword(),
+                user.isActive(),
```

Comment:
If you are not using these booleans (accountNonExpired, accountNonLocked, credentialsNonExpired, enabled) in your code, then you don't need the `isActive` field. But for now, you can leave them as they are.